teachertool/src/transforms/runEvaluate.ts

```diff
@@ -0,0 +1,17 @@
+import { getProjectMetaAsync, getProjectTextAsync } from "../services/BackendRequests";
+
+export async function runEvaluate(shareLink: string, rubric: string) {
```

Comment:
Per convention, append `Async` to async methods.

```suggestion
export async function runEvaluateAsync(shareLink: string, rubric: string) {
```